cogboard-webapp/src/components/widgets/types/PersonDrawWidget/index.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+
+import { TypographyVariant } from './styled';
+
+import storage from './helpers';
+import IconButton from '@material-ui/core/IconButton';
+import { Refresh } from '@material-ui/icons';
+import { getIsAuthenticated } from '../../../../selectors';
+import { useSelector } from 'react-redux';
+import { CenterWrapper } from '../TextWidget/styled';
+import { number, bool, string } from 'prop-types';
```

Comment:
For the imports we use the order: Libs and absolute imports; helpers utils redux actions etc; components